xmtp/src/client.rs

```diff
@@ -293,24 +293,21 @@ where
         &self,
         conn: &mut DbConnection,
         contact: &Contact,
-        // Message MUST be a pre-key message
-        message: &Vec<u8>,
+        prekey_message: PreKeyMessage,
```

Comment:
This is the easiest solution for now, since messages and invites are encoded differently. The caller is now responsible for passing in a decoded Prekey Message.

Required updates to [`process_inbound_invite`](https://github.com/xmtp/libxmtp/pull/185/files#diff-43fed88baca0e696206ee0e68177689c3149e6d0bbdd34fa8d8b4b9b53555271R332)